internal/brokers/manager.go

```diff
@@ -187,15 +187,15 @@ func (m *Manager) BrokerForSessionID(sessionID string) (broker *Broker, err erro
 	return broker, nil
 }
 
-// AbortSession sends a abortion request to the broker associated with the sessionID and then removes the
+// EndSession sends a abortion request to the broker associated with the sessionID and then removes the
```

Comment:
It’s not an abortion request anymore, it’s signaling the end of this session request.